ffv1.md

```diff
@@ -201,28 +203,41 @@ Background: a two's complement signed 16-bit signed integer was used for storing
 +---+---+---+---+
 ```
 
-The quantized sample differences L-l, l-tl, tl-t, t-T, t-tr are used as context:
+The quantized sample differences are used as context:
 
-PDF:$$context=Q_{0}[l-tl]+\left|Q_{0}\right|(Q_{1}[tl-t]+\left|Q_{1}\right|(Q_{2}[t-tr]+\left|Q_{2}\right|(Q_{3}[L-l]+\left|Q_{3}\right|Q_{4}[T-t])))$$
+PDF:$$context=Q_{0}[l-tl]+Q_{1}[tl-t]+Q_{2}[t-tr]+Q_{3}[L-l]+Q_{4}[T-t]$$
```

Comment:
There is abs() in the current RFC part, but I don't see what is absolute value of a table, and I don't see corresponding code in FFmpeg source code.
Note that the RFC version was containing multiplication too, and I don't see it too in FFmpeg source code.
See this [snipset](https://github.com/FFmpeg/FFmpeg/blob/74314f1f5f9ef69700eb18b85a8260e2754a31ef/libavcodec/ffv1_template.c#L32-L53).

I am not sure of it, help would be appreciated for confirmation.